R/error-checks.R

```diff
@@ -0,0 +1,27 @@
+#' Check if the provided object is of certain type
+#'
+#' @param object An object or a list of objects
+#' @param type String representation of the type that should be checked for
+#'
+#' @return TRUE if the object or all objects inside the list are of the given type.
+#' Only the first level of the given list is considered.
+#'
+#'
+isOfType <- function(object, type) {
+  object <- c(object)
+
+  return(all(
+    sapply(
+      object,
+      function(x) inherits(x, type)
+    )
+  ))
+}
+
+isSameLength <- function(...) {
+  args <- list(...)
+  return(length(
```

Comment:
This indentation is so hard to read. especially with the ==1 at the end. 
I am not sure why the styling allows this.  Can you make it one line?

Reply:
OK.